multi-node/aws/artifacts/scripts/install-controller.sh

```diff
@@ -10,22 +10,22 @@ export K8S_VER=v1.1.2
 # The CIDR network to use for pod IPs.
 # Each pod launched in the cluster will be assigned an IP out of this range.
 # Each node will be configured such that these IPs will be routable using the flannel overlay network.
-export POD_NETWORK=10.2.0.0/16
+export POD_NETWORK=
```

Comment:
the defaults are applied in the event that the yaml key either doesn't exist or is empty. you can define whatever you want there and it will get re-exported and overwritten regardless of the yaml key.